kubectl-pg/README.md

```diff
@@ -0,0 +1,64 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+This project uses Go Modules for dependency management to build locally
+Install go and enable go modules ```export GO111MODULE=on```
+
+As this project doesn't contain vendor directory. 
+
+Generate vendor directory using ```go mod vendor```
+
+Clone the kubectl pg plugin and build from the source using ```go install``` this will generate kubectl pg plugin's executable in ```go/bin/kubectl-pg.exe```
```

Comment:
Please update this instruction to have a step-by-step description similar to [this example](https://github.com/kubernetes/sample-cli-plugin#running). It will help potential users to start with the plugin faster